frontend/components/events/create/CustomTimeInput.tsx

```
@@ -18,10 +20,13 @@ export default function CustomTimeInput({
           handleChange(e.target.value);
         }}
+        ref={(input) => {
+          if (input) {
+            input.setCustomValidity(errorMessage || "");
+          }
+        }}
       ></input>
-      <label className="absolute -top-2 left-3 bg-white flex px-1 text-xs font-light">
-        {placeholder}
-      </label>
+      <label className="absolute -top-2 left-3 bg-white flex px-1 text-xs font-light">{placeholder}</label>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)